auth/build.gradle.kts

```
@@ -1,9 +1,13 @@
+@Suppress("DSL_SCOPE_VIOLATION")
 plugins {
   id("hedvig.android.library")
   id("hedvig.android.ktlint")
+  alias(libs.plugins.serialization)
 }
 
 dependencies {
   implementation(libs.koin.android)
   implementation(libs.okhttp.core)
+  implementation(libs.serialization.json)
+  implementation(projects.coreCommon)
```

I usually try to keep the `projects.` dependencies at the top and with a gap with the rest, so that at a glance you can see which other modules a module depends on. So smth like like 

```suggestion
  implementation(projects.coreCommon)

  implementation(libs.koin.android)
  implementation(libs.okhttp.core)
  implementation(libs.serialization.json)
```